static.config.js

```diff
@@ -576,32 +577,29 @@ const getPagesOfGuidesData = async client => {
 
   const pagesOfGuidesData = {};
 
-  allGuidePages.edges.map(guidePage => {
-    if (
-      guidePage.node.sections.length > 0 &&
-      guidePage.node.topics.edges.length > 0
-    ) {
-      const url =
-        '/' +
-        [
-          guidePage.node.topics.edges[0].node.topic.topiccollections.edges[0]
-            .node.topiccollection.theme.slug,
-          guidePage.node.topics.edges[0].node.topic.topiccollections.edges[0]
-            .node.topiccollection.slug,
-          guidePage.node.topics.edges[0].node.topic.slug,
-          guidePage.node.slug,
-        ].join('/');
+  allGuidePages.edges.map(async guidePage => {
+    if (guidePage.node.sections.length > 0) {
+      const guideUrl = await client.request(getPageUrlQuery, {
+        id: guidePage.node.id,
+      });
+      const url = guideUrl.allPages.edges[0].node.janisInstances[0].url;
```

Comment:
Choosing the first instance url is a good way to go. If I had to guess, I'd share your assumption that our previous url building method was chosen not out of a deliberate content strategy, but rather because it was impossible to get sensible janis url information easily.